checks/batch/__init__.py

```diff
@@ -313,7 +317,38 @@
                 ]
             }
         ]
-    }
+    },
+    {
+        'name': 'web_rpki',
+        'type': 'category',
+        'translation_key': 'siterpki',
+        'children': [
+            {
+                'name': 'web_rpki_exists',
+                'name_on_report': 'web_rpki_exists',
+                'type': 'test',
+                'translation_key': 'web rpki exists',
+            },
+            {
+                'name': 'web_rpki_valid',
+                'name_on_report': 'web_rpki_valid',
+                'type': 'test',
+                'translation_key': 'web rpki valid',
+            },
+            {
+                'name': 'ns_rpki_exists',
```

Comment:
This name needs to start with 'web_'; so 'web_ns_rpki_exists'. The name_on_report is fine.
```suggestion
                'name': 'web_ns_rpki_exists',
```
This is mainly when people want to merge web and email results from batch in a unified spreadsheet/dictionary.
The same for the rest of the names that I will notice below.